src/component/MovieDetails/MovieDetails.js

```diff
@@ -0,0 +1,34 @@
+import './MovieDetails.css';
+import searchLogo from '../../assets/search_logo.png';
+
+function MovieDetails(props) {
+    const commaSeparatedList = props.genres.join(', ');
+
+    return (
+        <div className = 'movie-details_outer-container'>
+            <div className='movie-details_title-row'>
+                <span className='movie-details_title'>netflixroulette</span>
```

Comment:
Move the string literals to constants file, this way you can use them in many places, if there is a change you can change at only one place

Reply:
Fixed as part of 5ea7447